packages/storybook-utils/src/storybook.utils.ts

```diff
@@ -43,22 +51,38 @@ export const createLayoutKnobs = (
   defaultValue: string = layoutOptions.vertical
 ) => select('Layout', layoutOptions, defaultValue, namespace);
 
-export const createTypographyKnobs = (namespace: string) => ({
-  fontSize: number('Font Size', 10, {}, namespace),
-  fontStyle: select(
-    'Font Style',
-    typographyOptions.fontStyle,
-    typographyOptions.fontStyle.normal,
-    namespace
-  ) as any,
-  fontWeight: select(
-    'Font Weight',
-    typographyOptions.fontWeight,
-    typographyOptions.fontWeight.normal,
-    namespace
-  ) as any,
-  fontColor: color('Font Color', colors.black['500'], namespace),
-});
+export const createTypographyKnobs = (
+  namespace: string,
+  fonts: string[] = []
+) => {
+  let fontsList: any = typographyOptions.fontFamily;
```

Comment:
We should avoid `any` type in codebase - please consider using `Record<string, string>`.